modin/core/storage_formats/base/query_compiler.py

```diff
@@ -1320,6 +1320,29 @@ def to_numeric(self, *args, **kwargs):  # noqa: PR02
         """
         return SeriesDefault.register(pandas.to_numeric)(self, *args, **kwargs)
 
+    @doc_utils.add_one_column_warning
+    @doc_utils.add_refer_to("to_timedelta")
+    def to_timedelta(self, *args, **kwargs):  # noqa: PR02
+        """
+        Convert argument to timedelta.
+
+        Parameters
+        ----------
+        unit : str
+            Denotes the unit of the arg for numeric arg. Defaults to "ns".
+        errors : {"ignore", "raise", "coerce"}
```

Comment:
I don't see those arguments in the function signature, it should error out in the docs

Reply:
Fixed.